lib/l10n/app_en.arb

```diff
@@ -196,6 +194,14 @@
   "readAll":"Read All",
   "emptyInbox":"Empty Inbox",
   "showAll":"Show All",
+  "gestures": "Gestures",
+  "navigation": "Navigation",
+  "sidebar": "Sidebar",
+  "fullScreenNavigationSwipeDescription": "Swipe anywhere to go back when left-to-right gestures are disabled",
+  "sidebarBottomNavSwipeDescription": "Swipe bottom nav to open sidebar",
+  "sidebarBottomNavDoubleTapDescription": "Double-tap bottom nav to open sidebar",
+  "commentSwipeGesturesHint": "Looking to use buttons instead? Enable them in the comments section in general settings.",
+  "postSwipeGesturesHint": "Looking to use buttons instead? Change what buttons appear on post cards in general settings.",
```

Comment:
Do these strings need to be added to the other .arb files?

Reply:
I did a test and it seems like its not needed. I'm also setting up Weblate for translations so it might be better to keep the other files empty! https://hosted.weblate.org/projects/thunder/thunder/

Reply:
Very nice!! Should we go through and remove any English strings in non-English files then?

Reply:
Yup, I think we can do that. We can also go ahead and remove any `.arb` files that have not yet been translated. This will just help the Weblate statuses better reflect the overall translation progress.

I've set up hooks so that Weblate automatically pushes new translations to the `translations` branch if you're interested. It seems like we already have a new language translated!

I'm holding off from making an announcement just yet until Weblate approves my application for libre (free) status on the project